driver/annotations.md

```diff
@@ -0,0 +1,171 @@
+# Adding UAST Annotations
+
+# Intro
+
+Once you have written the [AST parser](driver/sdk.md) the next step is to write
+the annotation code. This is the Go language code that will establish the rules to
+transform the original AST into the UAST and the implementation uses two types,
```

Comment:
Added "a process we call normalizing" so the user (that is learning) start to learn our terminology.